concepticondata/conceptlists/Verheyen-2019-1000.tsv

```diff
@@ -0,0 +1,1001 @@
+ID	NUMBER	DUTCH	ENGLISH	CONCEPTICON_GLOSS	CONCEPTICON_ID	AOA_MEAN	AROUSAL_MEAN	CONCRETENESS_MEAN	DOMINANCE_MEAN	FAMILIARITY_MEAN	IMAGEABILITY_MEAN	VALENCE_MEAN	NCHAR	SYLLABLE_LENGTH	BIGRAM	NEIGHBOR	FREQ_CELEX	FREQ_SUBTLEX	INSTRENGTH	BETWEENNESS
+Verheyen-2019-1000-1	1	aandachtig	attentive	ATTENTIVE	1908	7	4.9	4.2	4.05	5.45	4.85	4.9	10	3	85965	2	1136	108	0.792978114511445	7.7439781027361E-05
+Verheyen-2019-1000-2	2	aandrijvend	driving			11	5.47368421052632	3.5	4.5	2.5	2.9	4.47368421052632	11	3	123628	0	0	0	NaN	NaN
+Verheyen-2019-1000-3	3	aangenaam	pleasant			7.5	4.05	3.8	4.4	5.6	5	6.4	9	3	115185	0	1215	2834	6.42126594405105	0.000484524995441
+Verheyen-2019-1000-4	4	aangrijpend	evil	EVIL	45	11.25	5.45	4.2	5.1	4.85	4.95	4.45	11	3	122717	0	100	25	NaN	NaN
```

Comment:
aangrijpend is not evil, it is more like emotionally touching, moving, stirring – don’t know whether there is a corresponding concept?

Reply:
Can't think of one, but if missing leave it unlinked.

Remember that you can also use the app interface in your local machine (or the one we have online at  https://digling.org/calc/concepticon/ , which is a bit outdated but can still be used for most cases)